### ISSUE TITLE
[BUG] Dashy v2.1.2 consuming all assigned resources after changing config (either via GUI or direct conf.yml change)

### ISSUE THREAD

**Issue by CTHRU** [Lissy93/dashy#1500]
### Environment

Self-Hosted (Docker)

### System

raspberry pi os lite - 6.1.0-rpi8-rpi-v8 #1 SMP PREEMPT Debian 1:6.1.73-1+rpt1 (2024-01-25) aarch64 GNU/Linux

### Version

2.1.2

### Describe the problem

I'm still experiencing a similar issue to #1494 , but now it happens when dashy is active.

I started from the default config:
- I tried to add config to the `conf.yml` file directly (adding 1 section + 1 item).
- I tried to add a section with a single item via the GUI and `Save to Disk` from GUI.

It appears  that in both cases the symptoms are very similar to issue #1494 (due to repeating rebuild? But this time not at startup time but when the changes are applied while the app is running).
Observations:
- the docker container consumes all assigned CPU resources (in my case 50% op cpu)
- physical memory and swap memory are slowly filled until the system falls out of memory / the memory limit of the container is reached ( example below taken before the system would become unresponsive, but it will go up till 256 and filling swap memory too).   
 
      CONTAINER ID   NAME         CPU %     MEM USAGE / LIMIT   MEM %     NET I/O           BLOCK I/O         PIDS
                                  dashy        47.16%    205.7MiB / 256MiB   80.35%    396kB / 14.6MB    173MB / 1.2MB     84

- When the container is started the first time with the default conf.yml file, everything is fine and max memory usage does not exceed 150 MB.
- As soon as any change is applied while running, the above behavior is noticed.
- Keeping the config changes and bringing the docker container down and up as follows, solves the issue. The application seems to be correctly updated upon restart and then idles on cpu with about 150 MB RAM usage.

  ```sh
     sudo docker compose down
     sudo docker compose up -d --build
  ```
  This is also my current workaround to prevent the issue each time I apply a configuration change.

my `compose.yaml` file (fyi, i copied the `/app/public` folder contents of the source code to the mounted dir on the host)

```yaml
services:
  dashy:
    container_name: dashy
    hostname: dashy
    memswap_limit: 384M
    deploy:
      resources:
        limits:
          cpus: "0.50"
          memory: 256M
    restart: unless-stopped
    ports:
      - 80:80
    image: lissy93/dashy:latest
    volumes:
      - ${HOST_APP_DIR}/public:/app/public
    networks:
      - dashy
networks:
  dashy:
    name: dashy
```

### Please tick the boxes

- [X] You have explained the issue clearly, and included all relevant info
- [X] You are using a [supported](https://github.com/Lissy93/dashy/blob/master/.github/SECURITY.md#supported-versions) version of Dashy
- [X] You've checked that this [issue hasn't already been raised](https://github.com/Lissy93/dashy/issues?q=is%3Aissue)
- [X] You've checked the [docs](https://github.com/Lissy93/dashy/tree/master/docs#readme) and [troubleshooting](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#troubleshooting) guide ![important](https://img.shields.io/badge/Important!-F6094E)
- [X] You agree to the [code of conduct](https://github.com/Lissy93/dashy/blob/master/.github/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct)

**Comment by CrazyWolf13** [comment]
Noticing similar behaviour, today my container also entered a crashloop with the same js out of memory error like last time, however rebuilding and redeploying solved it temporarly.

**Comment by CrustyB** [comment]
Same here.  After a config change, the CPU spikes and dashy no longer works. Restarting the container does not bring dashy back.  Only a delete and rebuild brings dashy back online.

**Comment by charliegrahammm** [comment]
Having the same issue running on AWS ECS. Currently having to assign 1 vCPU & 2GB memory for it to successfully build using my config. Was running on 0.25 vCPU & 0.5GB memory previously. 

Navigating to the dashy webpage just shows - 
![image](https://github.com/Lissy93/dashy/assets/106349103/341837d0-7b95-4e26-95ac-559d390fe071)
then eventually fails with nothing in the console. 

Checking the logs shows the below - 
![image](https://github.com/Lissy93/dashy/assets/106349103/c2589339-7fab-4da8-9e2f-34cb5d2b60a7)

**Comment by CrustyB** [comment]
I reverted to 2.1.1 for the time being where this is not an issue.

**Comment by CrazyWolf13** [comment]
So did some testing, noticed dashy uses like 1.5GB to rebuild, when having unlimited ram(to 4GB) however then it's still not rebuilding correctly, it still exits with js heap allocation error.

But that could also be due to the raspberry pis memory management with docker in between.

**Comment by CrazyWolf13** [comment]
@CrustyB @charliegrahammm  @CTHRU  
Could you please test with the latest build from source and check if the issue persists, any feedback is appreciated, thanks!

**Comment by CTHRU** [comment]
@CrazyWolf13 saw your message and gave build from source a try. Unfortunately I'm not proficient in Node and Vue and I stumbled from missing dependency issues into directory rename issues trying to get Dashy built, before I could create a suitable docker image to be able to test. Hope someone else can help out with testing.

**Comment by CrazyWolf13** [comment]
@CTHRU Sure no problem, that was more meant if one is already running on build from source without docker.

In the meanwhile the docker image was updated, so you can just recreate the container and repull the :latest tag of the dashy image. Lmk if that fixes your issue.

**Comment by CTHRU** [comment]
@CrazyWolf13 I tried and pulled the latest docker image.
Unfortunately, now Dashy seems to loop on first start again? I tried both on my rpi, but because it's restricted in memory, I gave it a go on my desktop computer with 2GB of RAM for the container.

It took about 10 minutes for Dashy to become available, using 6 cores (50%) of the CPU. Max memory usage was about 1.4 GB.

I don't know if that is expected behavior. When I open the website during those first ten minutes, I get 'Possible error, check logs' message while it is busy.

In attachment are the stats of Docker Desktop.
![Dashy-24-03-27](https://github.com/Lissy93/dashy/assets/52134985/20b174df-7ba3-4984-ba8f-9e0fe1788eeb)

Just wanted to give you this feedback.

**Comment by charliegrahammm** [comment]
With 0.25vCPU & 0.5GB RAM (as I was previously using), I get the below in the logs: 

```
yarn run v1.22.19
$ NODE_OPTIONS=--openssl-legacy-provider npm-run-all --parallel build-watch start
$ vue-cli-service build --watch --mode production
$ node server
Checking config file against schema...
[1m[32m✔️ Config file is valid, no issues found[0m
SSL Not Enabled: Public key not present
[36m
 ██████╗  █████╗ ███████╗██╗  ██╗██╗   ██╗
 ██╔══██╗██╔══██╗██╔════╝██║  ██║╚██╗ ██╔╝
 ██║  ██║███████║███████╗███████║ ╚████╔╝
 ██║  ██║██╔══██║╚════██║██╔══██║  ╚██╔╝
 ██████╔╝██║  ██║███████║██║  ██║   ██║
 ╚═════╝ ╚═╝  ╚═╝╚══════╝╚═╝  ╚═╝   ╚═╝
[0m
[34m*******************************************************************************************
[0m[36mWelcome to Dashy! 🚀[0m
[32mYour new dashboard is now up and running with Docker
[34m*******************************************************************************************
[0m
Using Dashy V-2.1.2. Update Check Complete
[1m[32m✅ Dashy is Up-to-Date[0m
-  Building for production...
 WARN  A new version of sass-loader is available. Please upgrade for best experience.
<--- Last few GCs --->
[61:0x7f88bd7d96c0]   330204 ms: Mark-Compact 249.2 (258.1) -> 248.7 (258.1) MB, 3009.42 / 0.00 ms  (average mu = 0.510, current mu = 0.165) allocation failure; scavenge might not succeed
[61:0x7f88bd7d96c0]   333707 ms: Mark-Compact 249.9 (258.1) -> 249.7 (259.9) MB, 3494.86 / 0.00 ms  (average mu = 0.302, current mu = 0.002) allocation failure; scavenge might not succeed
<--- JS stacktrace --->
FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory
----- Native stack trace -----
error Command failed with signal "SIGABRT".
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ERROR: "build-watch" exited with 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

When giving it 1vCPU & 2GB RAM, the log looks like this: 

```
yarn run v1.22.19
$ NODE_OPTIONS=--openssl-legacy-provider npm-run-all --parallel build-watch start
$ vue-cli-service build --watch --mode production
$ node server
Checking config file against schema...
[1m[32m✔️ Config file is valid, no issues found[0m
SSL Not Enabled: Public key not present
[36m
 ██████╗  █████╗ ███████╗██╗  ██╗██╗   ██╗
 ██╔══██╗██╔══██╗██╔════╝██║  ██║╚██╗ ██╔╝
 ██║  ██║███████║███████╗███████║ ╚████╔╝
 ██║  ██║██╔══██║╚════██║██╔══██║  ╚██╔╝
 ██████╔╝██║  ██║███████║██║  ██║   ██║
 ╚═════╝ ╚═╝  ╚═╝╚══════╝╚═╝  ╚═╝   ╚═╝
[0m
[34m*******************************************************************************************
[0m[36mWelcome to Dashy! 🚀[0m
[32mYour new dashboard is now up and running with Docker
[34m*******************************************************************************************
[0m
Using Dashy V-2.1.2. Update Check Complete
[1m[32m✅ Dashy is Up-to-Date[0m
-  Building for production...
 WARN  A new version of sass-loader is available. Please upgrade for best experience.
 WARNING  Compiled with 2 warnings1:35:01 PM
 warning  
asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets: 
  fonts/Raleway-Variable.3b5635ac.ttf (302 KiB)
  js/chunk-204c2a20.615994a8.js (248 KiB)
  js/chunk-vendors.a867cf96.js (6.2 MiB)
  js/dashy.84e7ef3a.js (763 KiB)
 warning  
entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:
  dashy (7.18 MiB)
      css/chunk-vendors.d8067ad8.css
      js/chunk-vendors.a867cf96.js
      css/dashy.9e16d16f.css
      js/dashy.84e7ef3a.js
  File                                      Size             Gzipped
  dist/js/chunk-vendors.a867cf96.js         6346.96 KiB      2290.25 KiB
  dist/js/dashy.84e7ef3a.js                 763.29 KiB       230.66 KiB
  dist/js/chunk-204c2a20.615994a8.js        247.61 KiB       74.15 KiB
  dist/js/chunk-50f31ec3.0c5495f0.js        78.59 KiB        19.29 KiB
  dist/precache-manifest.e08ee60c2706744    19.48 KiB        4.49 KiB
  cc8168624cd5d8e84.js
  dist/js/chunk-180be55e.a602f1ad.js        15.94 KiB        5.30 KiB
  dist/js/chunk-468d3a74.a256f3b4.js        14.72 KiB        4.15 KiB
  dist/js/chunk-ae9287a0.246255c3.js        14.48 KiB        4.69 KiB
  dist/js/chunk-3b85e328.227556c9.js        14.08 KiB        4.17 KiB
  dist/js/chunk-2642eaf9.a6526654.js        13.53 KiB        4.54 KiB
  dist/js/chunk-7a79e1bb.5dc8c73c.js        12.29 KiB        4.24 KiB
  dist/js/chunk-7bba3126.e2ed5fba.js        11.40 KiB        3.99 KiB
  dist/js/chunk-08ca355a.1b5c2be7.js        10.26 KiB        3.79 KiB
  dist/js/chunk-05b33a20.b3dc2c02.js        10.22 KiB        4.12 KiB
  dist/js/chunk-460e6092.8f4150c5.js        10.16 KiB        3.73 KiB
  dist/js/chunk-445cc501.f09dc8f5.js        10.03 KiB        3.75 KiB
  dist/js/chunk-187213fc.8afd487d.js        9.94 KiB         3.69 KiB
  dist/js/chunk-0d10826c.cffb0d7e.js        8.36 KiB         2.96 KiB
  dist/js/chunk-326facd8.91bc8fb9.js        8.16 KiB         2.72 KiB
  dist/js/chunk-20c953a9.454143e4.js        8.11 KiB         2.91 KiB
  dist/js/chunk-93c6be8c.ae3c4ade.js        7.39 KiB         2.75 KiB
  dist/js/chunk-c8bd4cd0.151b6410.js        7.20 KiB         2.67 KiB
  dist/js/chunk-1ff5d837.c4aee8cb.js        7.15 KiB         2.68 KiB
  dist/js/chunk-74a1555a.c5fdb9b1.js        6.95 KiB         2.63 KiB
  dist/js/chunk-7a3f7db0.0147d53f.js        6.85 KiB         2.54 KiB
  dist/js/chunk-2f3fe870.cadd1ea7.js        6.71 KiB         2.31 KiB
  dist/js/chunk-070d32ac.a4ae2d9d.js        6.49 KiB         2.52 KiB
  dist/js/chunk-76101db0.70b00f21.js        6.24 KiB         2.35 KiB
  dist/js/chunk-e77c83e6.41ed4edc.js        6.19 KiB         2.30 KiB
  dist/js/chunk-92c623f0.000a873c.js        6.16 KiB         2.47 KiB
  dist/js/chunk-0c7116ec.6f1cabc9.js        6.02 KiB         2.40 KiB
  dist/js/chunk-4073bae0.7754034f.js        5.93 KiB         1.91 KiB
  dist/js/chunk-29548417.128162e9.js        5.71 KiB         2.25 KiB
  dist/js/chunk-370ed78e.1e37f5f2.js        5.39 KiB         2.14 KiB
  dist/js/chunk-1b35c628.c30996c5.js        5.34 KiB         2.16 KiB
  dist/js/chunk-8db027b8.8a8fd67b.js        5.29 KiB         2.10 KiB
  dist/js/chunk-15b37c0a.e9344e6b.js        5.24 KiB         2.11 KiB
  dist/js/chunk-e158ca64.6cae64f5.js        5.24 KiB         2.17 KiB
  dist/js/chunk-b7e4a5ce.17a5c8e0.js        5.23 KiB         2.22 KiB
  dist/js/chunk-28c92de0.04aa62bb.js        5.21 KiB         2.05 KiB
  dist/js/chunk-32eb6af1.dfd7e147.js        5.16 KiB         2.21 KiB
  dist/js/chunk-90c3e1c6.b2c27980.js        5.11 KiB         2.04 KiB
  dist/js/chunk-6f727b83.27b33359.js        5.11 KiB         1.90 KiB
  dist/js/chunk-47de4cfa.c83b4751.js        5.09 KiB         1.74 KiB
  dist/js/chunk-08fae180.c2121e47.js        4.99 KiB         2.05 KiB
  dist/js/chunk-4ab61964.044b317c.js        4.97 KiB         2.10 KiB
  dist/js/chunk-edd87180.e130bcb2.js        4.95 KiB         1.83 KiB
  dist/js/chunk-b52460ac.90e38546.js        4.88 KiB         2.03 KiB
  dist/js/chunk-cd40f4ae.12e7c601.js        4.83 KiB         1.99 KiB
  dist/js/chunk-0633ac20.c840e197.js        4.81 KiB         2.04 KiB
  dist/js/chunk-4ef6dcf5.98ccc823.js        4.81 KiB         2.06 KiB
  dist/js/chunk-832ca55a.4b0dd5ac.js        4.80 KiB         2.08 KiB
  dist/js/chunk-bd9012c4.fe314a88.js        4.52 KiB         1.96 KiB
  dist/js/chunk-da592926.ffde3fa2.js        4.47 KiB         1.90 KiB
  dist/js/chunk-2c65cac3.7d0acbc0.js        4.47 KiB         1.86 KiB
  dist/js/chunk-170af3bc.17674748.js        4.41 KiB         1.81 KiB
  dist/js/chunk-6b5de1e1.f84e935a.js        4.38 KiB         1.75 KiB
  dist/js/chunk-20bba19e.a87112e2.js        4.38 KiB         1.79 KiB
  dist/js/chunk-781da5fb.d0307276.js        4.32 KiB         1.45 KiB
  dist/js/chunk-4f2c58c5.b0ea83e0.js        4.17 KiB         1.64 KiB
  dist/js/chunk-736b2ef0.36231f88.js        4.17 KiB         1.72 KiB
  dist/js/chunk-f38e0ad2.ac83884b.js        4.15 KiB         1.79 KiB
  dist/js/chunk-3a3d0cd8.f3f3f2c8.js        4.13 KiB         1.80 KiB
  dist/js/chunk-674ac328.9a83caa4.js        4.12 KiB         1.88 KiB
  dist/js/chunk-cee89fa8.9fe9be9d.js        4.06 KiB         1.77 KiB
  dist/js/chunk-a235a3f6.e95536c4.js        4.06 KiB         1.59 KiB
  dist/js/chunk-c36eee8c.1b20d618.js        3.95 KiB         1.71 KiB
  dist/js/chunk-9221bf28.993b8333.js        3.77 KiB         1.67 KiB
  dist/js/chunk-c02e690a.288f88ff.js        3.65 KiB         1.57 KiB
  dist/js/chunk-14192a80.839bde78.js        3.64 KiB         1.51 KiB
  dist/js/chunk-b25c821e.bfab3021.js        3.47 KiB         1.55 KiB
  dist/js/chunk-b54d81ae.1d6a9a9f.js        3.45 KiB         1.52 KiB
  dist/js/chunk-7204030d.a48a0261.js        2.86 KiB         1.35 KiB
  dist/js/chunk-665a1900.2051f904.js        2.83 KiB         1.33 KiB
  dist/js/chunk-72e3b16c.b60be97b.js        2.80 KiB         1.30 KiB
  dist/js/chunk-195917c1.45d0f640.js        2.80 KiB         1.32 KiB
  dist/js/chunk-75cc9f4d.8f429d2d.js        2.74 KiB         1.30 KiB
  dist/js/chunk-0387fd77.83c8d9dd.js        2.74 KiB         1.28 KiB
  dist/js/chunk-0044633e.f4db6a47.js        2.40 KiB         1.11 KiB
  dist/js/chunk-73f090a0.7cf568e3.js        2.31 KiB         1.12 KiB
  dist/js/chunk-0c51289a.2a52d3de.js        2.27 KiB         1.06 KiB
  dist/js/chunk-372d1dd4.6756e844.js        2.25 KiB         1.07 KiB
  dist/js/chunk-7132ce43.fbc354a8.js        2.22 KiB         1.08 KiB
  dist/js/chunk-2d225b78.c20108bf.js        2.05 KiB         1.09 KiB
  dist/js/chunk-2ab49ff8.31979d5c.js        1.91 KiB         0.99 KiB
  dist/js/chunk-c0f28fc6.8bd24f8c.js        1.91 KiB         0.94 KiB
  dist/js/chunk-d42744f4.e6bba463.js        1.90 KiB         0.98 KiB
  dist/js/chunk-7795c4fe.2d4fc760.js        1.04 KiB         0.57 KiB
  dist/service-worker.js                    1.04 KiB         0.61 KiB
  dist/js/chunk-3767f013.0a2a7d30.js        0.75 KiB         0.45 KiB
  dist/css/dashy.9e16d16f.css               242.43 KiB       29.90 KiB
  dist/css/chunk-1ff5d837.91d9e1c7.css      11.54 KiB        1.75 KiB
  dist/css/chunk-ae9287a0.1671cdef.css      9.49 KiB         1.81 KiB
  dist/css/chunk-20c953a9.cfd7aba3.css      7.32 KiB         1.32 KiB
  dist/css/chunk-4073bae0.262be67e.css      5.86 KiB         1.00 KiB
  dist/css/chunk-0c7116ec.8d663b8e.css      3.98 KiB         0.96 KiB
  dist/css/chunk-29548417.1e586604.css      3.78 KiB         0.69 KiB
  dist/css/chunk-7795c4fe.8e5b7c8e.css      3.78 KiB         0.92 KiB
  dist/css/chunk-2642eaf9.103376cf.css      3.54 KiB         0.86 KiB
  dist/css/chunk-93c6be8c.b621be85.css      3.53 KiB         0.88 KiB
  dist/css/chunk-326facd8.fab73348.css      3.48 KiB         0.77 KiB
  dist/css/chunk-370ed78e.e42ddb3d.css      3.31 KiB         0.86 KiB
  dist/css/chunk-c8bd4cd0.25b1ca48.css      3.28 KiB         0.83 KiB
  dist/css/chunk-vendors.d8067ad8.css       2.74 KiB         0.83 KiB
  dist/css/chunk-28c92de0.04b75e3f.css      2.67 KiB         0.59 KiB
  dist/css/chunk-2f3fe870.d67cf285.css      2.51 KiB         0.64 KiB
  dist/css/chunk-7a79e1bb.6c6ecc47.css      2.48 KiB         0.67 KiB
  dist/css/chunk-204c2a20.9ea244f5.css      2.48 KiB         0.57 KiB
  dist/css/chunk-14192a80.31a5db2c.css      2.38 KiB         0.58 KiB
  dist/css/chunk-7a3f7db0.52735a01.css      2.38 KiB         0.57 KiB
  dist/css/chunk-74a1555a.8da9709e.css      2.30 KiB         0.56 KiB
  dist/css/chunk-7bba3126.b97a92c1.css      2.17 KiB         0.56 KiB
  dist/css/chunk-da592926.8c1925ff.css      2.06 KiB         0.49 KiB
  dist/css/chunk-781da5fb.38b3bad4.css      2.03 KiB         0.56 KiB
  dist/css/chunk-8db027b8.377fb75a.css      2.01 KiB         0.57 KiB
  dist/css/chunk-0d10826c.47d4c898.css      1.94 KiB         0.56 KiB
  dist/css/chunk-e77c83e6.729d6dc8.css      1.93 KiB         0.55 KiB
  dist/loading-screen.css                   1.93 KiB         0.65 KiB
  dist/css/chunk-90c3e1c6.5d01068d.css      1.86 KiB         0.50 KiB
  dist/css/chunk-08ca355a.0e2f8538.css      1.85 KiB         0.55 KiB
  dist/css/chunk-460e6092.0bcf49d9.css      1.85 KiB         0.56 KiB
  dist/css/chunk-e158ca64.b825db4a.css      1.81 KiB         0.55 KiB
  dist/css/chunk-070d32ac.3ca152a5.css      1.80 KiB         0.51 KiB
  dist/css/chunk-445cc501.d9af4531.css      1.79 KiB         0.53 KiB
  dist/css/chunk-187213fc.851bbb61.css      1.78 KiB         0.52 KiB
  dist/css/chunk-180be55e.2679cb7e.css      1.77 KiB         0.53 KiB
  dist/css/chunk-195917c1.8fa90971.css      1.65 KiB         0.42 KiB
  dist/css/chunk-2c65cac3.639a347e.css      1.64 KiB         0.51 KiB
  dist/css/chunk-32eb6af1.b73f2acc.css      1.59 KiB         0.48 KiB
  dist/css/chunk-4ab61964.950bd772.css      1.57 KiB         0.49 KiB
  dist/css/chunk-92c623f0.7601575f.css      1.55 KiB         0.47 KiB
  dist/css/chunk-b52460ac.d91d8d0b.css      1.49 KiB         0.47 KiB
  dist/css/chunk-4f2c58c5.e91567b0.css      1.38 KiB         0.43 KiB
  dist/css/chunk-a235a3f6.b5b0de24.css      1.36 KiB         0.41 KiB
  dist/css/chunk-6f727b83.40fd61fc.css      1.30 KiB         0.44 KiB
  dist/css/chunk-468d3a74.e7e4907a.css      1.26 KiB         0.42 KiB
  dist/css/chunk-05b33a20.60123b61.css      1.16 KiB         0.46 KiB
  dist/css/chunk-15b37c0a.ebae7724.css      1.15 KiB         0.36 KiB
  dist/css/chunk-3b85e328.8676c5ae.css      1.14 KiB         0.41 KiB
  dist/css/chunk-832ca55a.62100f79.css      1.12 KiB         0.36 KiB
  dist/css/chunk-0633ac20.857ad57c.css      1.04 KiB         0.34 KiB
  dist/css/chunk-bd9012c4.bbf2305d.css      1.04 KiB         0.34 KiB
  dist/css/chunk-170af3bc.5384b344.css      0.93 KiB         0.37 KiB
  dist/css/chunk-1b35c628.f7e5ac71.css      0.90 KiB         0.31 KiB
  dist/css/chunk-4ef6dcf5.f9dd4bd8.css      0.88 KiB         0.30 KiB
  dist/css/chunk-20bba19e.09a47e10.css      0.88 KiB         0.33 KiB
  dist/css/chunk-3a3d0cd8.5aaf7cba.css      0.88 KiB         0.33 KiB
  dist/css/chunk-edd87180.070a8cfa.css      0.87 KiB         0.35 KiB
  dist/css/chunk-b7e4a5ce.df4ad987.css      0.86 KiB         0.32 KiB
  dist/css/chunk-f38e0ad2.1ea48a31.css      0.84 KiB         0.28 KiB
  dist/css/chunk-76101db0.e1ddbf69.css      0.79 KiB         0.32 KiB
  dist/css/chunk-0387fd77.7aa83618.css      0.75 KiB         0.28 KiB
  dist/css/chunk-6b5de1e1.9eb66c9f.css      0.71 KiB         0.31 KiB
  dist/css/chunk-47de4cfa.8b8967c4.css      0.69 KiB         0.31 KiB
  dist/css/chunk-736b2ef0.98820bcd.css      0.60 KiB         0.33 KiB
  dist/css/chunk-c36eee8c.eccbbe45.css      0.59 KiB         0.23 KiB
  dist/css/chunk-372d1dd4.58ade778.css      0.46 KiB         0.24 KiB
  dist/css/chunk-08fae180.9b2da476.css      0.46 KiB         0.22 KiB
  dist/css/chunk-0c51289a.d6684378.css      0.38 KiB         0.17 KiB
  dist/css/chunk-7204030d.98a4aa99.css      0.36 KiB         0.16 KiB
  dist/css/chunk-75cc9f4d.98a4aa99.css      0.36 KiB         0.16 KiB
  dist/css/chunk-d42744f4.f1c873fc.css      0.36 KiB         0.16 KiB
  dist/css/chunk-674ac328.d604576c.css      0.36 KiB         0.19 KiB
  dist/css/chunk-2ab49ff8.2ca1d591.css      0.36 KiB         0.16 KiB
  dist/css/chunk-9221bf28.d7730813.css      0.31 KiB         0.20 KiB
  dist/css/chunk-c0f28fc6.b67ed63a.css      0.22 KiB         0.16 KiB
  dist/css/chunk-3767f013.c9ab3ab3.css      0.11 KiB         0.10 KiB
  dist/css/chunk-cee89fa8.0918bc41.css      0.08 KiB         0.10 KiB
  dist/css/chunk-b54d81ae.61a081a9.css      0.08 KiB         0.10 KiB
  dist/css/chunk-665a1900.eeb31e13.css      0.07 KiB         0.09 KiB
  dist/css/chunk-b25c821e.f58ec558.css      0.07 KiB         0.09 KiB
  dist/css/chunk-c02e690a.ccf83212.css      0.06 KiB         0.08 KiB
  dist/css/chunk-cd40f4ae.90cf07cd.css      0.06 KiB         0.07 KiB
  dist/css/chunk-0044633e.0e433876.css      0.00 KiB         0.02 KiB
  dist/css/chunk-7132ce43.0e433876.css      0.00 KiB         0.02 KiB
  dist/css/chunk-72e3b16c.0e433876.css      0.00 KiB         0.02 KiB
  dist/css/chunk-73f090a0.0e433876.css      0.00 KiB         0.02 KiB
  Images and other types of assets omitted.
 DONE  Build complete. Watching for changes...
```
The app starts successfully at this point. This is just starting the container without a custom config file applied. During this build process it eats all available CPU and memory.

**Comment by CrazyWolf13** [comment]
For me it used just very slightly more than 500MB depending on the size of the assets and config.

Also depending on the Hardware, dashy used a lot cpu, but that is what I expect for a rebuild.

So in general the errors may not even appear anymore, but the 0.25cpu and 0.5gm ram is just too lest.

As minimum dashy needs 1GB Ram.

**Comment by Lissy93** [comment]
Yeah, as @CrazyWolf13 said, the Node base image does require a bit more RAM, first build especially
This is something I'm working on (removing the need for rebuild, and eventually switching to a lighter base image)

**Comment by CrustyB** [comment]
As a pre-emptive action, I increased my docker instance from 2G to 4G of memory and increased the core count from 2 to 4.
I used the "latest" tag, rebuilt my docker container and all looked OK.
I opened the config, saved to disk and this is what I got:
Dashboard  | [OUTPUT] conf.yml file Changed, running build...
Dashboard  | 
Dashboard  | Error: ENOENT: no such file or directory, stat '/app/dist/index.html'
Dashboard  | Error: ENOENT: no such file or directory, stat '/app/dist/index.html'
Dashboard  | Error: ENOENT: no such file or directory, stat '/app/dist/index.html'
Dashboard  | Error: ENOENT: no such file or directory, stat '/app/dist/index.html'
Dashboard  | Error: ENOENT: no such file or directory, stat '/app/dist/index.html'
Dashboard  | Error: ENOENT: no such file or directory, stat '/app/dist/index.html'
Dashboard  | Error: ENOENT: no such file or directory, stat '/app/dist/index.html'

I set the tag back to 2.1.1 and await further tests.
